app/views/govuk_publishing_components/components/docs/date_input.yml

```diff
@@ -0,0 +1,54 @@
+name: Form date input
+description: Use the date input component to help users enter a memorable date or one they can easily look up.
+accessibility_criteria: |
+  Inputs in the component must:
+
+  * accept focus
+  * be focusable with a keyboard
+  * be usable with a keyboard
+  * be usable with touch
+  * indicate when they have focus
+  * be recognisable as form input elements
+  * have correctly associated labels
+  * be of the appropriate type for their use, e.g. password inputs should be of type 'password'
+
+  Labels use the [label component](/component-guide/label).
+
+  Avoid using autofocus and tabindex unless you have user research to support this behaviour.
+govuk_frontend_components:
+  - date-input
+examples:
+  default:
+    data: {}
+  with_name:
+    data:
```

Comment:
Is it worth adding a short description here to explain that the name is actually set to `name[field name]`?

Reply:
Good point. Description added explaining how we use this attribute.